src/tooltip.js

```diff
@@ -116,6 +117,15 @@ class Tooltip extends Component<Props, State> {
     };
   }
 
+  componentDidMount() {
+    if (this.state.isAnimating) {
+      InteractionManager.runAfterInteractions(() => {
+        this.measureChildRect();
+        this.setState({ isAnimating: false });
```

Comment:
```suggestion
        this.setState({ initialInteractionsComplete: true });
```

I would change this and **remove the if-statement**. We will always want to call this, even if there is no explicit animation. It will help ensure anything that needs to happen before we render the tooltip, does happen.